### PR TITLE
1 bit depth BMP support

### PR DESCRIPTION
**Pull request by whiite**
Hi! Thank you for helping out with Embedded Graphics development! Please:

- [x] Check that you've added passing tests and documentation
- [ ] Add a simulator example(s) where applicable
- [x] Add a `CHANGELOG.md` entry in the **Unreleased** section under the appropriate heading (**Added**, **Fixed**, etc) and appropriate crate (`embedded-graphics`, `simulator`, `tinytga`, `tinybmp`) if your changes affect the **public API**
- [x] Run `rustfmt` on the project
- [x] Run `./build.sh` (Linux/macOS only) and make sure it passes. If you use Windows, check that CI passes once you've opened the PR.

## PR description

[add your PR description here]
Added basic support for 1 bit depth BMP images to tinybmp.

### CODE REVIEW

File: tinybmp/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 ### Added
 
+- Added support for decoding 1 bit pixel depth BMP images.
```
Comment (rfuest) [inline tinybmp/CHANGELOG.md:15]: Please add an `###Added` header to the `[Unreleased]` section and move this there. And you should also add the PR number in front of it:
```suggestion
- #352 Added support for decoding 1 bit pixel depth BMP images.
```

File: tinybmp/src/lib.rs [hunk 2]
```diff
@@ -208,18 +205,23 @@ impl<'a> Iterator for BmpIterator<'a> {
...
-            // Shorter alias for easier to read code
-            let start = self.start_idx;
+            let byte_idx = self.bit_idx / 8;
 
             let pixel_value = match self.pixel_stride {
-                1 => u32::from(px[start]),
-                2 => u32::from_le_bytes([px[start], px[start + 1], 0, 0]),
-                3 => u32::from_le_bytes([px[start], px[start + 1], px[start + 2], 0]),
-                4 => u32::from_le_bytes([px[start], px[start + 1], px[start + 2], px[start + 3]]),
+                1 => (px[byte_idx] & 0b_1000_0000 >> self.bit_idx % 8 > 0) as u32,
```
Comment (rfuest) [inline tinybmp/src/lib.rs:211]: IMO this is a little hard to read. Can you change this into a block to make it more readable? Something like this:
```suggestion
                1 => {
                    let mask = 0b_1000_0000 >> self.bit_idx % 8;
                    (px[byte_idx] & mask != 0) as u32
                }
```

Comment (whiite) [reply tinybmp/src/lib.rs:211]: Thanks that is definitely better to read. I will add that in.

**Comment by rfuest** [comment]
LGTM! If @jamwaffles doesn't have anything to comment on this is ready to be merged.

**Comment by jamwaffles** [comment]
Just released in [0.2.3](https://crates.io/crates/tinybmp/0.2.3).

**Pull request merged**